demo/site/src/blocks/TextLinkBlock.tsx

```diff
@@ -7,6 +7,16 @@ import { LinkBlock } from "./LinkBlock";
 
 export const TextLinkBlock = withPreview(
     ({ data: { link, text } }: PropsWithData<DemoTextLinkBlockData>) => {
+        if (link.block && link.block.type === "damFileDownload" && "file" in link.block.props && link.block.props.file) {
+            return (
+                <Link data={link}>
+                    <>
+                        {text} ({Math.round(link.block.props.file.size / 1024)} KB)
```

Comment:
It would be cool if we had something like [PrettyBytes](https://github.com/vivid-planet/comet/blob/4a25d5e7bf0082b9dd15add7eabeaacae0897ec9/packages/admin/admin/src/helpers/PrettyBytes.tsx) here. Or at least support KB and MB since those are the realistic ones

Reply:
But since it's only in Demo, this doesn't block the merge

Reply:
Probably https://www.npmjs.com/package/filesize would be enough 🤔